runtime/calamari/src/fee.rs

```diff
@@ -128,9 +132,21 @@ mod multiplier_tests {
 			.max_total
 			.unwrap() - 10;
 
-		let base_fee = <Runtime as pallet_transaction_payment::Config>::WeightToFee::calc(
-			&frame_support::weights::constants::ExtrinsicBaseWeight::get(),
-		);
+		let remark = Call::System(frame_system::Call::<Runtime>::remark_with_event {
```

Comment:
just for my understanding, you used the remark extrinsic arbitrarily to get a small, constant size TX, right? or is there more to this choice?
maybe add the answer as a comment line to the code for future reference

Reply:
Yeah that's it, let me comment it.